src/services/WolfDen.Application/Requests/Queries/Attendence/AttendanceHistory/AttendanceHistoryQuery.cs

```diff
@@ -0,0 +1,13 @@
+﻿using MediatR;
+using WolfDen.Application.DTOs.Attendence;
+
+namespace WolfDen.Application.Requests.Queries.Attendence.AttendanceHistory
+{
+    public class AttendanceHistoryQuery:IRequest<List<WeeklySummaryDTO>>
```

Comment:
Purpose of this query

Reply:
To display the attendance history of employee (of chosen year)

Reply:
modified with month choosing and dailystatus filter